tests/bot/cogs/moderation/test_utils.py

```diff
@@ -276,6 +253,8 @@ def setUp(self):
 
                 if error:
                     self.ctx.bot.api_client.post.side_effect = ResponseCodeError(AsyncMock(), expected)
+                    err = self.ctx.bot.api_client.post.side_effect
+                    err.status = 400
```

Comment:
At the least, do this.
```suggestion
                    err = ResponseCodeError(Mock(status=400), expected)
                    self.ctx.bot.api_client.post.side_effect = err
```
You could also include this in the test case rather than using a boolean, which would make the code a tiny bit more direct.